src/Nest/Indices/Monitoring/IndicesStats/ShardCommit.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject]
+	public class ShardCommit
+	{
+		[JsonProperty("id")]
+		public string Id { get; set; }
+
+		[JsonProperty("generation")]
+		public int Generation { get; internal set; }
+
+		[JsonProperty("headers")]
+		public IDictionary<string, string> UserData { get; set; }
+
+		[JsonProperty("num_docs")]
+		public long TotalDocuments { get; internal set; }
```

Comment:
maybe `NumberOfDocuments`?

Reply:
Used same naming as `class Segment`

Reply:
`Segment` got it wrong IMO goes against `1-1` mapping. Its OK to expand abbreviations because abbreviations go against .NET standards. That `Segment` rename is probably on me sorry :)

Reply:
All good, renamed it.